tools/ui_integration/files/azure_billing_profile.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "bbc1684f-4b5e-45bf-8a50-64b029333731",
+  "profileName": "ui_integration_data_azure",
+  "biller": "direct",
+  "description": "Azure billing profile for UI integration test data",
+  "cloudPlatform": "azure",
+  "tenantId": "efc08443-0082-4d6c-8931-c5794c156abd",
+  "subscriptionId": "71d52ec1-5886-480a-9d6e-ed98cbf1f69f",
+  "resourceGroupName": "TDR",
+  "applicationDeploymentName": "shelby"
```

Comment:
This currently creates a billing profile in my application deployment on azure. We should an azure application deployment to use with UI integration tests. Or is there another good one to use that already exists and shouldn't get cleaned out?